backend/src/ee/services/permission/project-permission.ts

```diff
@@ -50,15 +50,17 @@ export enum ProjectPermissionIdentityActions {
   Create = "create",
   Edit = "edit",
   Delete = "delete",
-  GrantPrivileges = "grant-privileges"
+  GrantPrivileges = "grant-privileges",
+  AssumePrivileges = "assume-privileges"
```

Comment:
nit: how about calling this action `impersonate-privileges`? if we do that we might have to make other things consistent

Reply:
I think the wording is in correct right?  It's technically assuming the privileges right? Impersonation is more like when it's a user